### DISCUSSION TITLE
Import of backup data

### DISCUSSION THREAD

**Discussion by kabal00** [jonaswinkler/paperless-ng#980] [Support]
Dear Dev-Team,

I would like to tell you my complete backup story, because I have now some problems to restore my backup:

I backup'd all my data (from folder /var/lib/docker/volumes) from paperless-ng version 0.94

paperless_data
paperless_media
paperless_pgdata

After a upgrade to 1.32 I tried to restore this backup. It only works after I renamed the upper folders to 
paperless_ng_data
paperless_ng_media
paperless_ng_data

I copied everything to /var/lib/docker/volumes, installed the new paperless version and after that everythings runs perfect.

Now I had a datacrash and my HDD is defective. Luckily I did a backup some minutes ago. I installed on a completely new SSD an OS. After that I did the same procedure as described before but I have to rename the folders back to:

paperless_data
paperless_media
paperless_pgdata

After the installation of the latest paperless ng version everything works perfectly, except the original data could not be found. This means: I can access the server, I can see all data, categories, names, etc. I only can't open any PDF-documents. As soon as I click on a PDF file it shows no PDF view and I am not able to download it. It always give the error that the file could not be found.

Do you have an idea what I can do? It would be great to find a solution, because I have > 700 documents and your detection algorithm to give files automatically a tag is very very powerful!

Thanks in advance and have a nice day.

**Comment by jonaswinkler** [comment]
Restore the data to the same version you used to create the backup, verify that it works, then upgrade.

Upgrades also come with changes to the database schema, and your 0.9.4 data won't work in 1.3.2. Some of the versions also addressed some file naming issues.

I'll add that to the documentation.

**Reply by jonaswinkler** [reply]
Regarding the names of the docker volumes: there's a .env file in your paperless folder that defines the names of the docker volumes. If it's not there, the name of the parent folder is used. You're perfectly fine without this file, as long as the volumes are named correctly.
